src/Console/Commands/FixPermissionsCommand.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace CachetHQ\Cachet\Console\Commands;
+
+use DirectoryIterator;
+use Illuminate\Console\Command;
+use Illuminate\Support\Facades\Config;
+
+class FixPermissionsCommand extends Command
+{
+    /**
+     * The console command name.
+     *
+     * @var string
+     */
+    protected $name = 'cachet:chmod';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Fixes file permissions.';
+
+    /**
+     * Execute the console command.
+     *
+     * @return int|null
+     */
+    public function fire()
+    {
+        $this->recursiveChmod(app_path('storage').'/');
+
+        $database = Config::get('database.default');
+
+        if ($database === 'sqlite') {
+            chmod(app_path('database').'/', 755);
+            chmod(Config::get('database.sqlite.database'), 755);
```

Comment:
Don't read config here. Inject the actual result of config::get into the constructor.